infra/command/format

```diff
@@ -125,15 +128,21 @@ function check_python_files() {
   fi
 
   # Check python files
-  FILES_TO_CHECK_PYTHON=($(echo "${FILES_TO_CHECK[@]}" | tr ' ' '\n' | grep -E '\.py$'))
-  # Exceptional case: one-cmds don't have '.py' extension: ignore non-python source (cmake, etc) and ignore shell script: one-prepare-venv
-  FILES_TO_CHECK_PYTHON+=($(echo "${FILES_TO_CHECK[@]}" | tr ' ' '\n' | grep -E '^compiler/one-cmds/[^(\./)]*$' | grep -Ev '^compiler/one-cmds/one-prepare-venv$'))
-  # Exceptional case: onecc-docker don't have '.py' extension.
-  FILES_TO_CHECK_PYTHON+=($(echo "${FILES_TO_CHECK[@]}" | tr ' ' '\n' | grep -E '^compiler/onecc-docker/onecc-docker$'))
-  # Exceptional case: visq don't have '.py' extension.
-  FILES_TO_CHECK_PYTHON+=($(echo "${FILES_TO_CHECK[@]}" | tr ' ' '\n' | grep -E '^compiler/visq/visq$'))
-  # Exceptional case: fm-equalize doesn't have '.py' extension.
-  FILES_TO_CHECK_PYTHON+=($(echo "${FILES_TO_CHECK[@]}" | tr ' ' '\n' | grep -E '^compiler/fm-equalize/fm-equalize$'))
+  mapfile -t FILES_TO_CHECK_PYTHON < <(printf '%s\n' "${FILES_TO_CHECK[@]}" | grep -E '\.py$')
+
+  # Exceptional cases
```

Comment:
```suggestion
  # Exceptional cases: Some python files don't have '.py` extension.
```

Please keep the existing comment, which helped me understand the code.

Reply:
I've updated